remote-api/src/main/java/org/alfresco/rest/api/model/rules/RuleSet.java

```diff
@@ -37,10 +39,9 @@
 
     public static RuleSet of(String id)
     {
-        final RuleSet ruleSet = new RuleSet();
-        ruleSet.id = id;
-
-        return ruleSet;
+        return builder()
```

Comment:
Isn't a builder in this case (single field) an overkill? Or maybe other fields are coming in next PRs?

Reply:
This is meant for other fields coming in next PRs.